### PR TITLE
Adding bottom view binary tree problem solution

### PR DESCRIPTION
**Pull request by karimelazzouni**
- Adding and exporting a function that takes a `BinaryTree` as a parameter and returns the bottom view as an integer array 
- Solution based on https://www.geeksforgeeks.org/bottom-view-binary-tree/
- Modified `.gitignore` to ignore Visual Studio's files

### CODE REVIEW

**Comment by karimelazzouni** [comment]
Fixes #130

**Comment by karimelazzouni** [comment]
Fixes #120

**Comment by TheSTL** [comment]
Hi @karimelazzouni you can use `new Map()` instead of installing `hashmap` dependencies. In es6 `new Map()` is there to create Map object.

**Comment by karimelazzouni** [comment]
Hi @TheSTL , I implemented the changes. If you have no more comments let's merge this.

File: package.json [hunk 1]
```diff
@@ -20,5 +20,6 @@
     "eslint-config-airbnb-base": "^13.1.0",
     "eslint-plugin-import": "^2.14.0",
     "jest": "^25.0.0"
-  }
+  },
+  "dependencies": {}
```
Comment (TheSTL) [inline package.json]: Can you undo these changes?

File: src/_DataStructures_/Trees/BinaryTree/bottom-view-binary-tree/index.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    if (currentNode.leftChild != null && currentNode.leftChild.value != null) {
+      q.enqueue([currentNode.leftChild, currentHd - 1]);
+    }
+
+    if (currentNode.rightChild != null && currentNode.rightChild.value != null) {
+      q.enqueue([currentNode.rightChild, currentHd + 1]);
+    }
+  }
+
+  return hdToNodeValue.values();
```
Comment (TheSTL) [inline src/_DataStructures_/Trees/BinaryTree/bottom-view-binary-tree/index.js]: It should return an `array` instead of `Iterator ` @karimelazzouni
